CREATE TABLE produkt

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE produkt 
+(id_produktu bigserial PRIMARY KEY,
+ nazwa VARCHAR(50) UNIQUE NOT NULL,
+kategoria VARCHAR(50) NOT NULL,
+ilosc SMALLINT NOT NULL,
+cena NUMERIC(7,2) NOT NULL,
+rozmiar VARCHAR(20) NOT NULL,
+polozenie VARCHAR(20) NOT NULL,
+data_modyfikacji timestamp NOT NULL)
+
+
+CREATE TABLE zespol 
+(id_zespolu serial PRIMARY KEY,
+ kierownik VARCHAR(50) NOT NULL,

```

Comment:
I think 'kierownik' column shouldn't be simply a 'varchar' but instead should be an integer reference to the 'Pracownik' table, because kierownik will also be a pracownik